Hazelcast.Net/Hazelcast.IO.Serialization/IIdentifiedDataSerializable.cs

```diff
@@ -12,24 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-using System;
-
 namespace Hazelcast.IO.Serialization
 {
     /// <summary>
-    /// IdentifiedDataSerializable is an extension to
-    /// <see cref="IDataSerializable">IDataSerializable</see>
-    /// to avoid reflection during de-serialization.
-    /// Each IdentifiedDataSerializable is
-    /// created by a registered
+    /// IIdentifiedDataSerializable is a serialization method as an alternative to standard serialization.
+    /// It uses a factory to avoid reflection during deserialization.
     /// <see cref="IDataSerializableFactory">IDataSerializableFactory</see>
-    /// .
     /// </summary>
-    /// <seealso cref="IDataSerializable">IDataSerializable</seealso>
     /// <seealso cref="IPortable">IPortable</seealso>
     /// <seealso cref="IDataSerializableFactory">IDataSerializableFactory</seealso>
-    public interface IIdentifiedDataSerializable : IDataSerializable
```

Comment:
IDataSerializable is marked as Obsolete and is not functional, however IIdentifiedDataSerializable will work as before.